sdk/hdinsight/Microsoft.Azure.HDInsight.Job/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
-## 3.0.0-preview.1 (Unreleased)
-
+## 3.0.0-preview.2 (Unreleased)
+- Support using Apache Spark REST API to submit remote jobs to an HDInsight Spark cluster.
```

Comment:
>Support using Apache Spark REST API to submit remote jobs to an HDInsight Spark cluster. [](start = 2, length = 88)

I think that this description is more suitable for version preview.1.
I suggest to modify this description based what you have changed.